docs/root/operations/admin.rst

```diff
@@ -164,3 +164,13 @@ The fields are:
   Outputs /stats in `Prometheus <https://prometheus.io/docs/instrumenting/exposition_formats/>`_
   v0.0.4 format. This can be used to integrate with a Prometheus server. Currently, only counters and
   gauges are outputed. Histograms will be outputed in a future update.
+
+.. http:get:: /runtime
+
+  Outputs all runtime values on demand in a human-readable format. See
+  :ref:`here <config_runtime_v1>` for more information on how these values are configured
```

Comment:
Please link to this page: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/runtime, otherwise looks great!